buildSrc/build.gradle.kts

```diff
@@ -1,3 +1,8 @@
+/*
```

Comment:
seems like for build files copyright header is optional (my feeling based on kotlin repo)

Reply:
Doesn't harm and IntelliJ's default configuration decided to add the copyright, so probably its the easiest to keep it default 🙇